firefox-ios/nimbus-features/firefoxSuggestFeature.yaml

```
@@ -12,7 +12,25 @@ features:
           additional Search settings.
         type: Boolean
         default: false
+      available-suggestions-types:
+        description: >
+          A map of suggestion types to booleans that indicate whether or not the
+          provider should return suggestions of those types.
+        type: Map<SuggestionType, Boolean>
+        default: {
+          "amp": true,
+          "wikipedia": true,
+        }
```

In general, we prefer using the yaml native format for this stuff, instead of mixing & matching with json
```suggestion
        default:
          - amp: true,
          - wikipedia: true
```